kafka/client_async.py

```diff
@@ -201,18 +201,24 @@ def __init__(self, **configs):
             if key in configs:
                 self.config[key] = configs[key]
 
+        # these properties need to be set on top of the initialization pipeline
+        # because they are used when __del__ method is called
+        self._closed = False
+        self._wake_r, self._wake_w = socket.socketpair()
+        self._selector = self.config['selector']()
+
```

Comment:
Keep this newline as part of keeping the "required initialization section" separate

Reply:
👍